bundles/GenericExecutionEngineBundle/src/Messenger/Handler/AbstractAutomationActionHandler.php

```diff
@@ -151,17 +151,47 @@ protected function getCurrentJobStepConfig(GenericExecutionEngineMessageInterfac
         return $config;
     }
 
+    protected function getEnvironmentVariables(
+        GenericExecutionEngineMessageInterface $message
+    ): array {
+        $jobRun = $this->getJobRun($message);
+        $job = $jobRun->getJob();
+
+        return $job === null ? [] : $job->getEnvironmentData();
+    }
+
+    protected function replaceConfigValueWithEnvVariable(
+        string $value,
+        array $variables
+    ): mixed {
+        /** @var $matches array */
+        if(!preg_match_all("/pimcore_gee_get_env\('([^']*)'\)/", $value, $matches)) {
```

Comment:
why not just use `get_env`?

Reply:
Didnt want anyone to confuse it with maybe existing methods and be as descriptive as possible. But I am open to suggestions ofc, so if you prefer the shorter version, then that´s fine with me too.

Reply:
Internally discussed and settled for `job_env`